src/SFA.DAS.RoATPService.Application/AuditLogFieldComparison.cs

```diff
@@ -27,6 +27,25 @@ public async Task<IEnumerable<AuditLogEntry>> BuildListOfFieldsChanged(Organisat
                 }
             );
             ComparisonResult comparisonResult = organisationComparison.Compare(originalOrganisation, updatedOrganisation);
+            var updatedAt = updatedOrganisation.UpdatedAt;
+            if (!updatedAt.HasValue)
+            {
+                updatedAt = DateTime.Now;
+            }
+
+            var updatedBy = updatedOrganisation.UpdatedBy;
+            if (String.IsNullOrWhiteSpace(updatedBy))
+            {
+                updatedBy = "System";
+            }
+
```

Comment:
simplify above to:
     var updatedBy = string.IsNullOrWhiteSpace(updatedOrganisation.UpdatedBy) ? "System" : updatedOrganisation.UpdatedBy;